surround-lib/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyaml==17.12.1
+pylint==2.4.3
```

Comment:
Why does the Surround library require `pylint`?

Reply:
It is needed by CI

Reply:
Removed it